rsciio/tests/test_emd.py

```diff
@@ -21,13 +21,15 @@
 # National Lab (see https://emdatasets.com/ for more information).
 # NOT to be confused with the FEI EMD format which was developed later.
 
+import os
```

Comment:
```suggestion
```

as far as i can see, `os` is not used any more

Reply:
It is still line 124.

Reply:
Couldn't the same as in line 185 be used in line 124:
`hdf5_dataset_path = tmp_path / "test_dataset.emd"`

Reply:
Using that pattern doesn't work with the `setup_method` and the temporary director is required for the whole lifetime of  test class. They may be other way, but I didn't spend much effort on this, as this is secondary in this PR.

Reply:
OK, just thought we could get things working more similar with little effort